networks/moonriver.md

```diff
@@ -53,15 +77,15 @@ The tokens on Moonriver will also be called Moonriver (MOVR). Check out the Moon
 The smallest unit of Moonriver is called a Sediment. It takes 10^18 Sediment to make one Moonriver. The denominations are as follows:
 
 | Unit           | Moonriver            | Sediment                      |
-|----------------|----------------------|-------------------------------|
+|:--------------:|:--------------------:|:-----------------------------:|
 | Sediment       | 0.000000000000000001 | 1                             |
-| Kilosediment   | 0.000000000000001    | 1,000                         |
-| Megasediment   | 0.000000000001       | 1,000,000                     |
-| Gigasediment   | 0.000000001          | 1,000,000,000                 |
-| Micromoonriver | 0.000001             | 1,000,000,000,000             |
-| Millimoonriver | 0.001                | 1,000,000,000,000,000         |
+| KiloSediment   | 0.000000000000001    | 1,000                         |
+| MegaSediment   | 0.000000000001       | 1,000,000                     |
+| GigaSediment   | 0.000000001          | 1,000,000,000                 |
+| MicroMoonriver | 0.000001             | 1,000,000,000,000             |
+| MilliMoonriver | 0.001                | 1,000,000,000,000,000         |
 | Moonriver      | 1                    | 1,000,000,000,000,000,000     |
-| Kilomoonriver  | 1,000                | 1,000,000,000,000,000,000,000 |
+| KiloMoonriver  | 1,000                | 1,000,000,000,000,000,000,000 |
```

Comment:
made a similar comment before about gwei but I don't know if I have ever seen "GigaWei" always "Gigawei"... just an observation but I don't know if it really matters

Reply:
Yeah I changed this as well. Also, I think we'll use Kilosed instead if Kilosendiment (too long)